### ISSUE TITLE
Move travis private key to .ci folder

### ISSUE THREAD

**Issue by dessant** [kivy/kivy#5909]
`kivy/tools/travis/id_rsa.enc` should be moved to `.ci/id_rsa_travis.enc` to have these things in one place, and `.ci/id_rsa.enc` renamed to `.ci/id_rsa_appveyor.enc` for clarity.

**Comment by Julian-O** [comment]
I understand you want to move and rename [this file](https://github.com/kivy/kivy/blob/master/kivy/tools/travis/id_rsa.enc) to [this directory](https://github.com/kivy/kivy/tree/master/.ci).

I understand that some reference will need to be changed, and I don't know where that is.

Where is that second file?

**Comment by misl6** [comment]
We can likely remove everything, as we do not use travis anymore, and the linked key is not used for any of our integrations.

**Comment by misl6** [comment]
Yeah, definitely something we missed to remove via https://github.com/kivy/kivy/pull/6622

**Comment by AlexandreMJunior** [comment]
Hey, just wanna know if this issue is still needed...

**Comment by Julian-O** [comment]
> Hey, just wanna know if this issue is still needed...

Yep. Please just go ahead and submit a PR to remove it and reference this bug.

**Comment by AlexandreMJunior** [comment]
@Julian-O 

Subject: Clarification Needed: Travis CI and AppVeyor Configuration

I am currently working on this issue to move and rename specific CI-related keys (id_rsa.enc) into the newly created .ci directory for better organization. As part of this task, I need to confirm whether Travis CI and AppVeyor are still actively used within the project.

After reviewing the repository, I noticed the presence of the .travis.yml configuration file, but I am unsure if Travis CI remains a part of the active CI workflow or if the project has fully migrated to GitHub Actions. Additionally, I could not find an appveyor.yml file, which suggests AppVeyor may no longer be in use.

Could you please confirm:

Is Travis CI still in use, or should we consider deprecating the related configurations and keys?
Has AppVeyor been completely phased out, or is there any legacy dependency still tied to it?
Thank you in advance for your guidance.
